Sources/Algorithms/Unique.swift

```diff
@@ -15,18 +15,50 @@
 
 extension Sequence where Element: Hashable {
   /// Returns an array with only the unique elements of this sequence, in the
-  /// order of the first occurence of each unique element.
-  @inlinable
+  /// order of the first occurrence of each unique element.
+  ///
+  ///     let animals = ["dog", "pig", "cat", "ox", "dog", "cat"]
+  ///     let uniqued = animals.uniqued()
+  ///     print(uniqued)
+  ///     // Prints '["dog", "pig", "cat", "ox"]'
+  ///
+  /// - Returns: An array with only the unique elements of this sequence.
+  ///  .
+  /// - Complexity: O(*n*), where *n* is the length of the sequence.
   public func uniqued() -> [Element] {
-    uniqued(on: { $0 })
+    var seen: Set<Element> = []
+    var result: [Element] = []
+    for element in self {
+      if seen.insert(element).inserted {
+        result.append(element)
+      }
+    }
+    return result
   }
 }
 
 extension Sequence {
   /// Returns an array with the unique elements of this sequence (as determined
-  /// by the given projection), in the order of the first occurence of each
+  /// by the given projection), in the order of the first occurrence of each
   /// unique element.
-  @inlinable
```

Comment:
I don’t think this was rebased correctly. These annotations shouldn’t be deleted.